Lib/test/.ruff.toml

```diff
@@ -4,13 +4,13 @@ select = [
 ]
 extend-exclude = [
     # Failed to lint
-    "badsyntax_pep3120.py",
```

Comment:
Hmm, the pattern also matches without the `tokenizedata/` directory.

So we could use a list of filenames? I don't mind either way.

Reply:
I am in favor of being precise. It might be confusing for future readers to see here some files that are not really in the `Lib/test/` dir, but somewhere else.

Reply:
Oh, are these regexes?

We also could possibly exclude the whole `tokenizedata/` directory from this lint tbh. IIUC, we don't actually have any tests there; those are just data files to be read by the test suite.

Reply:
Not regex, but based on glob patterns:

https://docs.astral.sh/ruff/settings/#exclude

Okay, shall we keep the subdirs and exclude the whole `tokenizedata/` directory?

Reply:
> Okay, shall we keep the subdirs and exclude the whole `tokenizedata/` directory?

Yes, that sounds good to me! So we'd just have one entry for the `tokenizedata/` directory: `tokenizedata/*.py`